src/Resources/app/administration/src/module/sw-mail-template/component/sw-mail-template-list/sw-mail-template-list.html.twig

```diff
@@ -0,0 +1,94 @@
+{% block sw_mail_template_list_grid %}
+<sw-card
```

Comment:
arent there better blocks to not copy the entire grid?

Reply:
Unfortunatly there is no block. Do you know any other solution instead of an pr to shopware itself?

https://github.com/shopware/platform/blob/trunk/src/Administration/Resources/app/administration/src/module/sw-mail-template/component/sw-mail-template-list/sw-mail-template-list.html.twig